src/components/NewMessage/NewMessageTypingIndicator.vue

```diff
@@ -139,11 +139,10 @@ export default {
 
 <style lang="scss" scoped>
 .indicator {
-	position: absolute;
-	bottom: calc(100% - 2em);
-	left: 0;
 	width: 100%;
 	padding-right: 12px;
+	padding-left: 12px;
+	margin-bottom: 12px;
```

Comment:
```suggestion
	margin-bottom: 4px;
```

Smaller offset seems also good for, and as benefit, we save some pixels for the chat itself